tests/virt_autotest/virt_utils.pm

```diff
@@ -81,20 +81,39 @@ sub locate_sourcefile {
 }
 
 sub get_repo_0_prefix {
-    # Get customized repo location from REPO_0_PREFIX and append missing forward slash to the end
+    my ($default_repo_urls_ref) = @_;
+    my @default_repo_urls = @{$default_repo_urls_ref};
     my $repo_0_prefix = get_var("REPO_0_PREFIX", "");
     $repo_0_prefix .= '/' unless $repo_0_prefix =~ /\/$/;
-    $repo_0_prefix = ($repo_0_prefix ne "/" ? $repo_0_prefix : "http://openqa.suse.de/assets/repo/");
-    return $repo_0_prefix;
+    foreach my $url (@default_repo_urls) {
+        my $repo_0 = get_var("REPO_0");
```

Comment:
Is this function used by arm tests only? I don't know much about the current tests deployed in apac2 openqa server, I just would like to let you be aware that `REPO_0` has not been a mandatory setting for tests with iPXE boot. Arm tests have not this issue as they still boot from PXE.